assets/css/base/_base.scss

```diff
@@ -22,3 +24,45 @@ h3 {
 p {
   font-size: 1em;
 }
+
+// Links and Buttons
+a {
+  color: $blue-dark;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+a:hover,
+a:focus {
+  background-color: $orange-focus;
+}
+
+button {
+  background-color: $green-light;
+  border: 2px solid $blue-alt-light;
+  border-radius: 70px;
+  font-size: 24px;
```

Comment:
It's a great idea to use a [relative unit](https://hankchizljaw.com/wrote/relative-sizing-with-em-units/) for the font size here. If you look at Figma you can see that the `<h3>` style is `24px` which is equal to `1.5em`. You can figure out a font size in `em` by dividing the font size by the base font size (which is 16):

24px / 16px = 1.5em

Reply:
got it